src/containers/EventForm.js

```diff
@@ -34,14 +39,32 @@ export const withCreateEvent = compose(
   })
 );
 
+export const withDelete = compose(
+  getContext({ firebase: PropTypes.object }),
+  withHandlers({
+    onConfirm: ({ item, onCancel }) => () => {
+      // TODO sent to firebase / call EventCRUD prop
+      console.log("DELETE EVENT", item);
+      onCancel(); // close modal after delete
+    }
+  })
+);
+
 export const withEventActionState = compose(
   connect(selectAuth),
   withState("createEventVisible", "setCreateEventVisibility", false),
   withState("editEvent", "setEditEvent", null),
+  withState("confirmDeleteEvent", "setConfirmDelete", null),
   withHandlers({
     toggleCreateEvent: ({
       createEventVisible,
       setCreateEventVisibility
-    }) => () => setCreateEventVisibility(!createEventVisible)
+    }) => () => setCreateEventVisibility(!createEventVisible),
+    toggleConfirmDeleteModal: ({
+      confirmDeleteEvent,
+      setConfirmDelete
+    }) => eventToDelete => {
+      setConfirmDelete(eventToDelete);
+    }
```

Comment:
Aa, det låter bra! Japp så kan man länka rätt in på update och även klara en reload etc :+1: